dev/io.openliberty.transport.http_fat/publish/servers/ContentTypeResponseHeader/server.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+    Copyright (c) 2025 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<server description="Test content-type in response headers.">
+
+    <include location="../fatTestCommon.xml"/>
+
+    <httpEndpoint id="defaultHttpEndpoint"
```

Comment:
Why not just include fatTestPorts.xml?  https://github.com/OpenLiberty/open-liberty/blob/integration/dev/fattest.simplicity/autoFVT-defaults/fatTestPorts.xml